docs/changes_0.x.rst

```diff
@@ -8,7 +8,7 @@ Version 0.30.0 (2016-01-23)
 
 Compatibility notes:
 
-- you may need to use -v (or --info) more often to actually see output emitted
+- you may need to use -v (or --info) more often to see output actually emitted
```

Comment:
I understand the emphasis intended. The latter - the problem that splitting the infinitive creates here is that it creates the impression that "actually seeing" - the user's vision, (or the appearance of something to the user's vision?) - somehow changes. What is actual? Both the output emitted and what the user sees as a result :)

It seems that what you are trying to draw attention to is the contrast, the change as a result of adding `-v`.

It could be the result of this chain of simplification: "In actual fact, you can see it" --> "Actually, you can see it" --> "You can actually see it" --> "to actually see it". 

Personally, I think this one functions equally well without "actually" at all. Although for the writing to be a bit more vigorous, one could say: "...more often to be able to see output emitted". This avoids doing something like "to see, actually, ..."

Reply:
Maybe the difference comes from programmer's perspective:

In the code, if I do logger.info("hey"), I would say I logged/emitted "hey".

But, whether the user can actually see "hey" on their screen (== whether it actually ends up on their screen) depends on the log level chosen. By default (WARNING), there will be no output, but with -v (INFO), it will be output.

That's what I intentioned to document here.

Reply:
So the result is interesting :)

I rewrote.

Reply:
Does this match what you wanted to express?:

```
- use -v (or --info) multiple times to be able to see log level
  INFO messages (the output default is messages of log level WARNING).
```

Reply:
It's an old changelog entry, so hard to remember, but I would assume that "more often" meant "more frequently" and not "multiple times" (at once, in one borg invocation).

Reply:
more frequently does make logical sense to me...

Reply:
does not make..... 🤦

Reply:
back then at v0.30 I guess we changed the default log level from INFO to WARNING.

before 0.30, borg was rather verbose by default (so you didn't need -v that often).

after the change (and that's what this log entry is about), borg has been more silent by default (like many unix tools are). so if you missed some output you previously had, it advises you to use -v more frequently than before 0.30.